packages/template-retail-react-app/config/default.js

```diff
@@ -45,7 +46,9 @@ module.exports = {
             // This differs from the siteId in commerceAPIConfig for testing purposes
             siteId: 'aaij-MobileFirst',
             isProduction: false
-        }
+        },
+        // getAppOrigin will take x-forwarded-host header into account if this is true
```

Comment:
@hajinsuha1 Make sure you are syncing with develop. We have removed this config in the develop